obspy/core/tests/test_waveform_plugins.py

```diff
@@ -81,7 +81,10 @@ def test_read_and_write(self):
                     tr.stats.location = "00"
                     tr.stats.channel = "EHE"
                     tr.stats.calib = 0.199999
-                    tr.stats.delta = 0.005
+                    if format == 'AH':
```

Comment:
AH uses floats not double for the delta, therefore the test fails using a delta of 0.005